database/migration/migrateData.ts

```diff
@@ -12,24 +12,15 @@ import { getAppEntryPointDir } from '../../src/components/utilityFunctions'
 const FUNCTIONS_FILENAME = '43_views_functions_triggers.sql'
 const INDEX_FILENAME = '44_index.sql'
 
-const { version, isProductionBuild } = config
+const { version } = config
 const isManualMigration: Boolean = process.argv[2] === '--migrate'
 const simulatedVersion: string | undefined = process.argv[3]
 
 const migrateData = async () => {
   let databaseVersion: string
 
-  const appVersion = !isProductionBuild
-    ? // In development, pretend the version is one higher than it is,
-      // so we can keep getting changing migrations when testing
-      semverInc(version, 'patch')
-    : version
-
   try {
     databaseVersion = (await DB.getDatabaseVersion()).value
-    // No migration if database version matches current version, but we still
-    // proceed if this is a manual migration
-    if (semverCompare(databaseVersion, appVersion) >= 0 && !isManualMigration) return
```

Comment:
No longer checking in order to avoid whole script -- the script will run regardless, and we'll be solely relying the version as to what actually gets run.

But the important thing is that `43_views_functions_triggers.sql` gets run after every snapshot install, which will make sure the generated columns get fixed every time.